pkg/apis/networking/types.go

```diff
@@ -77,6 +86,26 @@ type NetworkPolicyIngressRule struct {
 	From []NetworkPolicyPeer
 }
 
+// NetworkPolicyEgressRule describes a particular set of traffic that is allowed out of pods
+// matched by a NetworkPolicySpec's podSelector. The traffic must match both ports and to.
```

Comment:
Small nit: `The traffic must match both Ports and To`

Reply:
@cmluciano this comment is still pending it looks like